shlr/bochs/src/libbochs.c

```diff
@@ -217,6 +247,79 @@ bool bochs_open(libbochs_t* b, const char * rutaBochs, const char * rutaConfig)
 			}
 		}
 	}
+#else
+	#define PIPE_READ 0
+	#define PIPE_WRITE 1
+	int aStdinPipe[2];
+	int aStdoutPipe[2];
+	int nChild;
+	int nResult;
+	char *newargv[] = { rutaBochs, "-q","-f", rutaConfig, NULL };
```

Comment:
const char vs char issue here.. it will be easier for you to use execl than execv